cpu/src/csr.rs

```diff
@@ -0,0 +1,25 @@
+// Licensed under the Apache-2.0 license
+
+#[cfg(feature = "riscv")]
```

Comment:
Could you link to the relevant VeeR manual sections in comments in this file to make it easier for folks reading this code?